Sources/CryptomatorCloudAccess/Box/BoxCloudProvider.swift

```diff
@@ -579,7 +579,7 @@ public class BoxCloudProvider: CloudProvider {
 
 	private func convertStandardError(_ error: Error) -> Error {
 		switch error {
-		case let error as BoxAPIError where error.responseInfo.statusCode == 401:
+		case let error as BoxAPIError where error.responseInfo.statusCode == 400 || error.responseInfo.statusCode == 401:
```

Comment:
This mapping seems too "broad" to me. "400" only means that it's a "Bad Request". It doesn't have to be an unauthorized error. But it can be? To be honest, I don't even know. How did you find this out?

Reply:
You’re right! I’ve specified the error handling to better differentiate between different cases. I discovered this by analyzing the error response from AppAuth: the -10 code specifically corresponds to an “invalid_grant” error, which indicates that the token has expired or been revoked. I’ve refactored this in commit 0c1a348.